01-landing-page/scss/style.scss

```diff
@@ -0,0 +1,286 @@
+//Basic styles 
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+//Variables
+
+$dark-blue: #283C52;
+$green: #219A9C;
+$light-gray: #E9EAF1;
+
+// html {
+//     font-size: 10px;
+// }
+
+body {
+    font-family: 'Raleway', sans-serif; //300, 400, 500
+    font-size: 10px;
+    color: $dark-blue;
+    overflow-x: hidden;
+}
+
+li {
+    list-style: none;
+}
+
+.container {
+    width: 90%;
+    margin: auto;
+}
+
+.d-flex {
+    display: flex;
+}
+
+//N A V B A R 
+
+.nav {
+    .d-flex {
+        height: 6rem;
+        justify-content: space-between;
+        align-items: center;        
+    }
+}
+
+.nav__logo {
+    font-size: 2rem;
+    font-weight: 500;
+    position: relative;
+    .trademark {
+        display: inline-block;
+        transform: scale(.6);
+        position: absolute;
+        left: 6rem;
+        top: -.2rem;
+    }
+}
+
+
+.nav__list__item{
+    margin-left: 2.5rem;
+    font-size: 1.1rem;
+}
+
+.logo__link,
+.nav__link {
+    text-decoration: none;
+    color: $dark-blue;
+    &:hover {
+        color: $green;
+    }
+}
+
+.menu__toggler {
+    display: none;
+}
+
+.menu__toggler__icon {
+    display: none;
+}
+
+//H E R O 
+
+.hero {
+    height: 80vh;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 1;
+    grid-template-areas:
+    'text img';
+}
+
+.hero__copy {
+    grid-area: 'text';
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding-top: 4rem;
+}
+
+.hero__heading {
+    font-size: 3rem;
+    font-weight: 300;
+    padding-bottom: 3rem;
+}
+
+.hero__subheading__text {
+    font-size: 1.3rem;
+    padding-bottom: 5rem;
+}
+
+.btn {
+    background-color: $green;
+    color: white;
+    font-size: .9rem;
+    display: inline-block;
+    width: 10rem;
+    padding: 1rem 1.5rem;
+    border: none;
+    &:hover {
+        background-color: lighten($color: $green, $amount: 5);
+    }
+}
+
+.hero__img {
+    margin-top: 3rem;
+    grid-area: 'img';
```

Comment:
Something's odd with this `grid-area` (not working properly). When using it you don't need to write with `'`.

```scss
// doesn't work
grid-area: 'img';

// it works
grid-area: img;
```

You can notice the incorrect CSS in the dev tools: 
![image](https://user-images.githubusercontent.com/14869087/89895837-5d1db880-dbd4-11ea-8a5e-64f165f225b3.png)